### ISSUE TITLE
Food recall warning: Extra tab order between thumbnail image links

### ISSUE THREAD

**Issue by sarasuri** [mmiron-HC/WST-RSAMS#108]
URL: https://hcdev.openplus.ca/en/alert-recall/certain-happy-veggie-world-hearts-choices-and-vegefarm-brand-vegan-and-vegetarian-1
User Agent: Firefox (v92), Chrome (v94), MS Edge
Language: English and French

Issue Description: There is an extra tab order between each thumbnail image link and last focusable element and the first image link and last image link and "Next" link. It could be because of the tabindex=0 specified in the <div> element, which contains the image link.

![image](https://user-images.githubusercontent.com/12256067/137014121-1a6350f3-8a37-4014-94bf-5ec2b25df18d.png)

`<div class="slick__slide slide slide--0 slick-slide slick-current slick-active" data-slick-index="0" aria-hidden="false" style="width: 170px;" tabindex="0"><div class="views-field views-field-field-product-image__revision-id"><div class="field-content"><a href="https://hcdev.openplus.ca/sites/default/files/styles/x_large/public/alert/recall/60175/Happy%20Veggie%20World%20Crispy%20Chicken%20Patty%20%28simulated%20meat%29%2C%203%20kg%20-%20Label_1.JPG?itok=igEPzq28" class="wb-lbx lbx-image wb-init wb-lbx-inited" tabindex="0" id="wb-auto-2"><img class="thumbnail img-responsive" src="/sites/default/files/styles/medium/public/alert/recall/60175/Happy%20Veggie%20World%20Crispy%20Chicken%20Patty%20%28simulated%20meat%29%2C%203%20kg%20-%20Label_1.JPG?itok=uijToQV_" loading="lazy" typeof="foaf:Image" alt="Happy Veggie World Crispy Chicken Patty (simulated meat), 3 kg - Label" width="187" height="220"></a></div></div></div>`

Steps to reproduce: 

1. Open the URL provided above
2. Navigate the page using the "Tab" key

Expected results: There shouldn't be extra tab order between the elements mentioned above.

**Comment by echouin** [comment]
Hi @mmiron-HC please note that this is a test build of a recall that was completed by one of our users practicing. I will be asking OP to verify on this as it looks to be a function issue.

**Comment by echouin** [comment]
@mmiron-HC from OP: we can't fix this one at this point. What's happening is slick, the module that does the slider, adds a tab index then WxT is adding the second one. We don't want to get into altering upstream code unless absolutely necessary and this issue is flagged as moderate.

**Comment by echouin** [comment]
@sarasuri from OP: Issue has been resolved and can no longer replicate.

**Comment by hydegeoff** [comment]
Failed.  I have to press the tab-key twice when going from one product image to the next.  I should only need to do it once. 
If the previous arrow button is active the cursor will not navigate to it.

Here is an example article:
https://recalls-rappels.canada.ca/en/alert-recall/certain-stellar-bay-shellfish-brand-oysters-recalled-due-norovirus

**Comment by echouin** [comment]
@hydegeoff I noticed this was closed?

**Comment by hydegeoff** [comment]
Clicked the wrong button

**Comment by echouin** [comment]
@hydegeoff, reported to Drupal. From OP: Reported back to the Slick module maintainers.
﻿
﻿May be tracked here: ﻿https://www.drupal.org/project/slick/issues/3273815

**Comment by hydegeoff** [comment]
It looks like the developers are using this Splide carousal, can you confirm what version version of Splide you have implemented in RSAMS, and is it the same version installed in both prod and DEV?  Is the same version mentioned within this Drupal project tacker: https://www.drupal.org/project/slick/issues/3273815

**Comment by mmiron-HC** [comment]
@zimw7 @YuxuanXie01 @40999221 Can you retest and see if this is still an issue?

**Comment by 40999221** [comment]
Retested against PROD. Issue no longer exist, Splide carousal has been removed. now, thumbnail image links can be access with one tab. 

![image](https://github.com/user-attachments/assets/66a5535e-db30-464b-8759-56fe6ba212ee)

**Comment by zimw7** [comment]
Issue no longer exists. The splide carousel has been removed.
![image](https://github.com/user-attachments/assets/f9c68665-fb18-46ac-a535-02c1c4f1c733)

**Comment by YuxuanXie01** [comment]
Retested, issue resolved since the thumbnail image no longer utilizes sliders or carousels for its layout.
